modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -6,36 +6,25 @@
 [id="virt-checking-cluster-dpdk-readiness_{context}"]
 = DPDK checkup
 
-Use a predefined checkup to verify that your {product-title} cluster node can run a virtual machine (VM) with a Data Plane Development Kit (DPDK) workload with zero packet loss. The DPDK checkup runs traffic between a traffic generator pod and a VM running a test DPDK application.
+Use a predefined checkup to verify that your {product-title} cluster node can run a virtual machine (VM) with a Data Plane Development Kit (DPDK) workload with zero packet loss. The DPDK checkup runs traffic between a traffic generator and a VM running a test DPDK application.
 
 You run a DPDK checkup by performing the following steps:
 
-. Create a service account, role, and role bindings for the DPDK checkup and a service account for the traffic generator pod.
-. Create a security context constraints resource for the traffic generator pod.
+. Create a service account, role, and role bindings for the DPDK checkup.
 . Create a config map to provide the input to run the checkup and to store the results.
 . Create a job to run the checkup.
 . Review the results in the config map.
 . Optional: To rerun the checkup, delete the existing config map and job and then create a new config map and job.
 . When you are finished, delete the DPDK checkup resources.
 
 .Prerequisites
-* You have access to the cluster as a user with `cluster-admin` permissions.
 * You have installed the OpenShift CLI (`oc`).
-* You have configured the compute nodes to run DPDK applications on VMs with zero packet loss.
-
-[IMPORTANT]
-====
-The traffic generator pod created by the checkup has elevated privileges:
-
-* It runs as root.
-* It has a bind mount to the node's file system.
-
-The container image of the traffic generator is pulled from the upstream Project Quay container registry.
-====
+* The cluster is configured to run DPDK applications.
```

Comment:
Duplicated line.

Reply:
This change is from PR #62804 that this PR depends on.
The lines are not duplicated (one line talks about the cluster and another on the project).

Reply:
You are correct.
Sorry.